### PR TITLE
update nodes to support new features

### PR DESCRIPTION
**Pull request by DiegoAntonino**

### CODE REVIEW

**Comment by jtheoof** [comment]
Not related to this MR, but it would be nice if the code could pass our `mnubo` `eslint` config. https://github.com/mnubo/eslint-config-mnubo

File: nodes/mnubo/config/mnubo-utils.js [hunk 1]
```diff
@@ -107,15 +149,14 @@ function GetNewMnuboClient(mnuboconfig) {
...
-   //console.log('client=',client);
    return client;
 }
 exports.GetNewMnuboClient = GetNewMnuboClient;
 
 
 
-//var debug = true;
-var debug = false;
+var debug = true;
```
Comment (DiegoAntonino) [inline nodes/mnubo/config/mnubo-utils.js]: Done!

File: nodes/mnubo/events/mnubo-events.js [hunk 2]
```diff
@@ -4,35 +4,42 @@ module.exports = function(RED) {
...
-      if (return_promise==1)
+      var client = ConfigMnuboUtils.GetNewMnuboClient(thisNode.mnuboconfig);
+      var options = {
+                  'reportResults': thisNode.reportResults,
+                  'objectsMustExist': thisNode.objectsMustExist
+      };
+
+      try {
+          if (typeof(msg.payload) == 'string') {
+                  msg.payload = JSON.parse(msg.payload)
```
Comment (DiegoAntonino) [inline nodes/mnubo/events/mnubo-events.js]: Done!

File: nodes/mnubo/events/mnubo-events.js [hunk 3]
```diff
@@ -42,63 +49,56 @@ module.exports = function(RED) {
...
       return_promise = return_promise || 0;
       
-      var client = ConfigMnuboUtils.GetNewMnuboClient(thisNode.mnuboconfig);      
-      
-      //ConfigMnuboUtils.DebugLog("msg.payload=",msg.payload);    
-      //ConfigMnuboUtils.DebugLog("typeof(msg.payload)=",typeof(msg.payload));    
-      try{
+      var client = ConfigMnuboUtils.GetNewMnuboClient(thisNode.mnuboconfig);
+      var options = {
+                  'reportResults': thisNode.reportResults,
```
Comment (DiegoAntonino) [inline nodes/mnubo/events/mnubo-events.js]: Done!

File: nodes/mnubo/objects/mnubo-objects.js [hunk 4]
```diff
@@ -41,62 +67,60 @@ module.exports = function(RED) {
...
-         if (myArray.length !=2) {
-            ConfigMnuboUtils.UpdateStatusErrMsg(thisNode,"bad amount of arguments");
-            return;
-         }
+      var client = ConfigMnuboUtils.GetNewMnuboClient(thisNode.mnuboconfig);
+
+      try {
+          if (typeof(msg.payload) == 'string') {
+                  msg.payload = JSON.parse(msg.payload)
+                }
```
Comment (DiegoAntonino) [inline nodes/mnubo/objects/mnubo-objects.js]: Done!

File: nodes/mnubo/package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
     "version"      : "1.1.1",
```
Comment (DiegoAntonino) [inline nodes/mnubo/package.json]: Done!

**Pull request merged**